dist/angular-patternfly.js

```diff
@@ -2767,6 +2767,345 @@ angular.module('patternfly.views').directive('pfDataList', [
+ * <li>.multiSelect            - (boolean) Allow multiple tile selections, selectItems must also be set, not applicable when dblClick is true. Default is false
+ * <li>.selectionMatchProp     - (string) Property of the items to use for determining matching, default is 'uuid'
+ * <li>.selectedItems          - (array) Current set of selected items
+ * <li>.checkDisabled          - ( function(item) ) Function to call to determine if an item is disabled, default is none
+ * <li>.onCheckBoxChange       - ( function(item) ) Called to notify when a checkbox selection changes, default is none
+ * <li>.onSelect               - ( function(item, event) ) Called to notify of item selection, default is none
+ * <li>.onSelectionChange      - ( function(items) ) Called to notify when item selections change, default is none
+ * <li>.onClick                - ( function(item, event) ) Called to notify when an item is clicked, default is none
+ * <li>.onDblClick             - ( function(item, event) ) Called to notify when an item is double clicked, default is none
+ * </ul>
+ *
+ * @param {Array} items the data to be shown in the data tiles<br/>
+ *
+ * @example
+ <example module="patternfly.views" deps="patternfly.utils">
+ <file name="index.html">
+   <style>
+   hr {
+          display: block;
+          height: 10px;
+          border: 0;
+          border-top: 1px solid #525252;
+          margin: 1em 0;
+          padding: 0;
+        }
+   </style>
+   <div ng-controller="ViewCtrl" class="row" style="display:inline-block; width: 100%;">
+     <div class="col-md-12">
+       <div pf-data-tiles id="exampleDataTiles" config="config" items="items">
+         <div class="col-md-12">
+           <span>{{item.name}}</span>
+         </div>
+         <div class="col-md-12">
+           <span>{{item.address}}</span>
+         </div>
+         <div class="col-md-12">
+           <span>{{item.city}}, {{item.state}}</span>
+         </div>
+       </div>
+     </div>
+     <hr class="col-md-12">
+     <div class="col-md-12">
+       <form role="form">
+         <div class="form-group">
+           <label>Selection</label>
+           </br>
+           <label class="radio-inline">
+             <input type="radio" ng-model="selectType" value="checkbox" ng-change="updateSelectionType()">Checkbox</input>
+           </label>
+           <label class="radio-inline">
+             <input type="radio" ng-model="selectType" value="tile" ng-change="updateSelectionType()">Tile</input>
+           </label>
+           <label class="radio-inline">
+             <input type="radio" ng-model="selectType" value="none" ng-change="updateSelectionType()">None</input>
+           </label>
+         </div>
+       </form>
+     </div>
+     <div class="col-md-12">
+       <form role="form">
+         <div class="form-group">
+           <label class="checkbox-inline">
+             <input type="checkbox" ng-model="config.dblClick">Double Click</input>
```

Comment:
It may make sense to `ng-hide` this (or `ng-disabled`) if "Tile" isn't selected above as it's somewhat confusing that you can check "Double Click" and there is no difference if "Checkbox" or "None" is selected.